### DISCUSSION TITLE
GitHub Pages stuck on Certificate Request Error

### DISCUSSION THREAD

**Discussion by brandoncamerer** [community/community#43506] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

GitHub Pages stuck on Certificate Request Error

All users are not getting HTTPS...

<img width="881" alt="image" src="https://user-images.githubusercontent.com/50874245/211066532-db2b21a6-c610-4f67-932b-6414448b6696.png">

**Comment by radar-blip** [comment]
If this is happening to multiple users, maybe a [Bug-Report](https://support.github.com/contact/bug-report) is in order.

**Comment by brandoncamerer** [comment]
You need to point your www CNAME to username.github.io. 

Not in the docs :(
